app/views/vm_common/_reconfigure.html.haml

```diff
@@ -92,6 +92,8 @@
                           "selectpicker-for-select-tag" => "")
             - if @cores_options.length <= 1
               @cores_options[0]
+            %span.help-block{"ng-if" => "angularForm.cores_per_socket_count.$dirty"}
+              =_("Note: a restart of the virtual machine might be reuired for the changes to apply.")
```

Comment:
You have a typo here: reuired -> required

Reply:
Oh thank you for catching this!
Added a fix here: https://github.com/ManageIQ/manageiq-ui-classic/pull/3317